distributions/startupway/src/server/index.ts

```diff
@@ -1,10 +1,10 @@
 import "./script"
 import { Server } from "@startupway/main/lib/server";
-import "@startupway/users/lib/server";
 import "@startupway/database/lib/server";
+import "@startupway/users/lib/server";
 import path from "path";
 import express from "express";
-let server = Server.getInstance();
+const server = Server.getInstance();
 server.registerRouterUI (express.static (path.resolve(__dirname, "../ui")));
 server.start();
```

Comment:
```suggestion
await server.start();
```
Si asta e functie async, deci si asta trebuie pusa cu await.

Reply:
Ai facut asta?